lib/validator/validator.go

```diff
@@ -162,10 +188,12 @@ func validateProtocol(v *validator.Validate, structLevel *validator.StructLevel)
 	// names.
 	if num, err := p.NumValue(); err == nil {
 		if num == 0 {
-			structLevel.ReportError(reflect.ValueOf(p.NumVal), "Protocol", "protocol", "protocol number invalid")
+			structLevel.ReportError(reflect.ValueOf(p.NumVal),
+				"Protocol", "", reason("protocol number invalid"))
```

Comment:
for my understanding, why did you change `"protocol"` field to `""` in this one and all the following `ReportError`?